test/provisioner/provision_ibmcloud_initializer.go

```diff
@@ -206,8 +208,14 @@ func initProperties(properties map[string]string) error {
 		}
 	}
 
-	if err := initVpcV1(); err != nil {
-		return err
+	if len(IBMCloudProps.ApiKey) > 0 {
+		if err := initClustersAPI(); err != nil {
+			return err
+		}
```

Comment:
Why was this moved out of the `if strings.EqualFold(needProvisionStr` block? Doesn't it mean that `TEST_PROVISION` will be ignored and the cluster will be created whenever the api key is set?

Reply:
So I think I've got this wrong after chatting to Jordan & Tia - this doesn't trigger the cluster creation, but just gets ready for it and it's needed for both provisioning and deprovisioning, so there is a change of behaviour, but it's because the deprovisioning would have failed previously which is a bit strange.

I think we basically want to get to the point that the provisioner-cli doesn't require any particular value of `TEST_PROVISION`, or `TEST_TEARDOWN` as we are explicitly calling those actions

Reply:
In chatting through with Tia, she has tested that the provision and deprovision actions work irrespective of the `TEST_PROVISION`, or `TEST_TEARDOWN` values. I think there is probably some clarification which could be done more generally to improve the interaction between the environment variables and actions, but that is the outside the scope of this PR, so this conservation can be resolved